frontend/util.lua

```diff
@@ -127,18 +127,19 @@ function util.secondsToClock(seconds, withoutSeconds)
         end
     else
         local round = withoutSeconds and require("optmath").round or passthrough
-        local hours = string.format("%02d", seconds / 3600)
+        local days = string.format("%02d", seconds / (24*3600)) --implicit math.floor for string.format
```

Comment:
(nit: space after `--` to make it more readable - otherwise I may think you're doing C and decrementing the variable `--implicit` :)